web3-python-app/README.md

```diff
@@ -0,0 +1,10 @@
+# web3-python-app
+
+This example showcases resolving Bitwise's ETHW ENS addresses using [web3.py](https://github.com/ethereum/web3.py).
+
+## Developing
+
+```
+pip install -r requirements.txt
```

Comment:
have you ever tried `uv`? my friend is building it, apparently much faster than `pip` or anything out there right now: https://github.com/astral-sh/uv

Reply:
Wow first time I heard of it but those stats look incredible! Will try that out the the lint tool `ruff` under the same umbrella

Reply:
Yep felt like magic! Love that it also makes virtualenv easier! Also documented that in the README!

Reply:
LFG! 

he's a good follow on twitter: https://x.com/charliermarsh